apps/src/templates/codeReviewGroups/Student.jsx

```diff
@@ -34,7 +33,8 @@ export default function Student({followerId, name, index}) {
               display: 'flex'
             }}
           >
-            {name}
+            <DragHandle {...provided.dragHandleProps} />
```

Comment:
Am I understanding correctly that you can only drag via click on the six dot thing now?

Reply:
And doesn't regress any of the keyboard drag and drop (which is already a little broken)?

Reply:
Yep, you can only drag if you click on the drag handle (six dots). And yep, keyboard drag and drop functions the same (can drag within a group but not to another group). There's a highlight around the drag handle now too -
![Screen Shot 2021-11-10 at 11 01 23 AM](https://user-images.githubusercontent.com/85528507/141176568-6eb6026b-6707-47e7-97f8-f0084efa2c90.png)